src/components/Home/ClusterDashboardItem.js

```diff
@@ -143,10 +144,12 @@ function ClusterDashboardItem({
   dispatch,
   nodePoolsLoadError,
   clusterId,
+  releases,
 }) {
   // If the cluster has been deleted using gsctl, Happa doesn't know yet.
   if (!cluster) return null;
 
+  const release = releases[cluster.release_version] ?? null;
```

Comment:
Maybe the errors in the tests are related to this line. `null` does not support being accessed (Line 257 - 263)

Reply:
So what should I do?

Reply:
You already solved this by adding `?` to the mentioned lines